content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+
+![Prioritized Team User Story Backlog](/images/outcomes/application-development/backlog.png)
+
+Finally, a user story provides a common way to talk about the **goals, motivations, and definitions of success** of your end users. By carefully crafting user stories based on an understanding of your users, you are more likely to build something that delivers value to them. 
+
+
+### Structure of a User Story
+
+Write user stories in a consistent manner that provides the team with enough context. 
+
+Common parts of a user story include:
+
+#### Title
+
+The title is a short description that explains the content of the story at a glance and is written from the perspective of a target user. The user might be a real human, a generalized persona, or even another software system.
+
+{{% callout %}}
+Example:
+
+_A Sales rep can download a sales proposal as a PDF_
+{{% /callout %}}
+
+#### Description
+
+The main description of a story should explain which type of user wants the feature, what it allows that user to do, and what outcome or benefit this brings about. An established format is:
+
+AS A `type of user`
+
+I WANT `some goal`
+
+SO THAT `benefit for the end user`
+
+{{% callout %}}
+Example:
+
+**AS A** _sales rep_
+
+**I WANT** _to download a PDF version of a sales proposal_
+
+**SO THAT** _I can email it to a prospective customer_
+{{% /callout %}}
+
+#### Acceptance Criteria
+
+Acceptance criteria list the scenarios the product manager uses to verify that the story is  complete. A common format is the [Behavioral Specification](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) format, also called the [Gherkin](https://en.wikipedia.org/wiki/Cucumber_(software)#Gherkin_language) language:
+
+GIVEN `situation or context`
+
+WHEN `some action is carried out`
+
+THEN `some consequences`
+
+
+{{% callout %}}
+Example:
+
+**GIVEN** _I visit the proposal summary page_
+
+**WHEN** _I click the button labeled “PDF Download”_
+
+**THEN** _a PDF file of the sales proposal downloads to my computer_
+{{% /callout %}}
+
+Acceptance criteria are important because they **clearly describe when the story is _done_**. Without an explicit statement of “done”, the feature risks [scope creep](https://en.wikipedia.org/wiki/Scope_creep). In the above example, the feature is done when the sales proposal  downloads to the user's computer as a PDF file. 
+
+Would it be handy if the user could email the PDF directly from the product? Sure!
+
+Would it be great if the system would automatically prompt the user to follow up on the sale proposal in one week? Heck yeah!
```

Comment:
First sentence: it is "sale" proposal or "sales" proposal?
Second sentence: Add a comma between "Heck" and yeah.

Reply:
Done